src/MoBi.Presentation/Tasks/Interaction/AbstractStartValuesTask.cs

```diff
@@ -166,10 +166,16 @@ protected bool HasEquivalentFormula(IStartValue startValue, IFormula targetFormu
          return _startValuePathTask.HasEquivalentFormula(startValue, targetFormula);
       }
 
+      protected static bool HasEquivalentStartValue(IStartValue startValue, IParameter parameter)
+      {
+         var (value, _) = parameter.TryGetValue();
+         return HasEquivalentStartValue(startValue, value);
```

Comment:
This was causing the crash before. Sometimes, we can have formula that cannot be evaluated and it was crashign the code. I am using our new TryGetValue here in the overload